DataStream.js

```diff
@@ -19,20 +19,20 @@ DataStream = function(arrayBuffer, byteOffset, endianness) {
     this.buffer = new ArrayBuffer(arrayBuffer || 1);
   }
   this.position = 0;
-  this.endianness = endianness == null ? DataStream.LITTLE_ENDIAN : endianness;
+  this.endianness = endianness === null ? DataStream.LITTLE_ENDIAN : endianness;
```

Comment:
This is using the == comparison for a reason. Missing function args are undefined, which doesn't === null, but does == null.